test/import/FSHErrorListener.test.ts

```diff
@@ -303,20 +303,4 @@ describe('FSHErrorListener', () => {
       /Mapping rules must include at least one space both before and after the '->' operator.*File: Invalid\.fsh.*Line: 7\D*/s
     );
   });
-
-  // ########################################################################
-  // # Missing space after *                                                #
-  // ########################################################################
-
-  it('should make sense of errors due to no space after * in a rule', () => {
```

Comment:
Oh for there to have been a reasoning. The reasoning was that it just started working with the grammar changes, and I can't entirely explain why. If anyone else can tell why from the grammar changes, I am interested to know.

Reply:
Hmmm... I kind of like requiring that space.  Does that make me weird?  Or mean?

Reply:
I'm not sure. I was under the impression it was basically a bug that that whitespace was required. But a bug we couldn't really figure out how to fix. SUSHI should've been whitespace insensitive if it wanted to truly implement FSH. Obviously that requirement is going to have to change in FSH since whitespace before rules will now be meaningful. But it doesn't really make sense to me to just require whitespace in our actual grammar just because it looks better (and I do agree it looks better).

Reply:
Especially if we intend to address the other whitespace issues that cause problems for people. Specifically related to whitespace around `=`.

Reply:
But...
<img width="889" alt="image" src="https://user-images.githubusercontent.com/2278253/115713980-9c480000-a344-11eb-8b18-dfdaf4230043.png">
http://hl7.org/fhir/uv/shorthand/reference.html#rules-and-rule-order

Reply:
I'd bet you $1 that line was more driven by how SUSHI worked than SUSHI conforming to that line, but I surrender 🏳️